dev-python/openpyxl/openpyxl-2.6.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="3640394bff97"
+
+DESCRIPTION="Pure python reader and writer of Excel OpenXML files"
+HOMEPAGE="https://openpyxl.readthedocs.io/en/stable/"
+SRC_URI="https://bitbucket.org/${PN}/${PN}/get/${PV}.tar.bz2 -> ${P}.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND="
+	dev-python/jdcal[${PYTHON_USEDEP}]
+	dev-python/et_xmlfile[${PYTHON_USEDEP}]
+"
+DEPEND="
+	test? (
```

Comment:
This can also go into BDEPEND.

Reply:
I'm not sure about moving them all to `BDEPEND`, as for example in [this](https://bitbucket.org/openpyxl/openpyxl/src/default/openpyxl/tests/conftest.py) file, there is `import pytest`, so I think it used in guest python environment.

Reply:
Technically, all testing will most likely be done using host environment, since the guest env could involve some C extensions that won't work on host. However, at the moment we do not support cross properly in Python, so it's not very important.